src/components/ProgramsTable/ProgramsTable.jsx

```diff
@@ -1,37 +1,103 @@
 import React, { Component } from 'react';
+import PropTypes from 'prop-types';
 import classNames from 'classnames';
 import { navigate } from 'gatsby';
+import { StatusAlert } from '@edx/paragon';
+import { faExclamationTriangle } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 
 import Layout from '../Layout/Layout';
 
 class ProgramsTable extends Component {
   constructor(props) {
     super(props);
-    this.state = {
-      programs: [
+    const programData = this.props.programQueryData;
```

Comment:
nit: Could rewrite this as `const { programQueryData: programData } = this.props;`